README.md

```diff
@@ -38,6 +38,12 @@ git submodule update --init --recursive
 
 Then set up [git secrets](#git-secrets) and fire up the [development servers](#running-the-dev-servers). Optionally, you can [set up your Intellij](https://docs.google.com/document/d/1DtESBapEzvuti7xODTFPHorwmLM7LybF-6D5lhbIkLU/edit) for UI or API work.
 
+Before doing any development, you must run:
+```Shell
+./project.rb appengineRun
```

Comment:
`appengineRun` is a gradle command, not project.rb.  I'd update it to `./gradlew compileGeneratedJava appengineRun` to include the Swagger generation step as well.

I can't comment about MapStruct because I'm not familiar.